vizro-core/src/vizro/actions/_actions_utils.py

```diff
@@ -37,14 +37,13 @@ def _get_component_actions(component) -> List[Action]:
 
 
 def _validate_selector_value_NONE(value: Union[SingleValueType, MultiValueType]) -> ValidatedNoneValueType:
-    validated_value: ValidatedNoneValueType = value
-    if value == [NONE_OPTION]:
-        validated_value = [None]
-    elif value == NONE_OPTION:
-        validated_value = None
-    elif isinstance(value, list) and len(value) > 1 and NONE_OPTION in value:
-        validated_value = [i for i in value if i != NONE_OPTION]  # type: ignore[assignment]
-    return validated_value
+    if value == NONE_OPTION:
+        return None
+    elif value == [NONE_OPTION]:
+        return [None]
+    elif isinstance(value, list):
+        return [i for i in value if i != NONE_OPTION] or [None]  # type: ignore[list-item, return-value]
```

Comment:
I believe the `value == [NONE_OPTION]` case is now contained in the one that follows? So probably not required as a separated `elif` unless you think it's clearer this way.

Reply:
Hm it is clearer, but I think we can skip it, makes it shorter which is good :)